Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -1238,7 +1330,7 @@ theorem MemLp.of_nnnorm_le_mul {f : α → E} {g : α → F} {c : ℝ≥0} (hg :
   ⟨hf, (eLpNorm_le_nnreal_smul_eLpNorm_of_ae_le_mul hfg p).trans_lt <|
       ENNReal.mul_lt_top ENNReal.coe_lt_top hg.eLpNorm_lt_top⟩
 
-theorem MemLp.of_enorm_le_mul [TopologicalSpace ε']
```

Comment:
Side note: this and all other removals are because of the `variable` line added in line 960.